home/templates/home/question_detail.html

```diff
@@ -1,6 +1,7 @@
 {% extends "home/base.html" %}
```

Comment:
I tried to count that each div has a closing div and something doesn't add up correctly.

Reply:
I double-checked. it's probably the closing </form> tag